easybuild/easyconfigs/l/libsigc++/libsigc++-3.4.0-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'ConfigureMake'
+
+name = 'libsigc++'
+version = '3.4.0'
+
+homepage = 'https://libsigcplusplus.github.io/libsigcplusplus/'
+description = """The libsigc++ package implements a typesafe callback system 
+for standard C++."""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = [FTPGNOME_SOURCE]
+sources = [SOURCELOWER_TAR_XZ]
+checksums = ['02e2630ffb5ce93cd52c38423521dfe7063328863a6e96d41d765a6116b8707e']
+
+builddependencies = [
+    ('binutils', '2.38'),
+    ('Graphviz', '5.0.0'),
+    ('libxslt', '1.1.34'),
+    ('mm-common', '1.0.5'),
+    ('CMake', '3.24.3'),
```

Comment:
It requires CMake but it's a ConfigureMake project? And i would also expect it to still require Autotools since we run autogen. I.e. i would expect it to require the identical list of builddeps as before.